workflow/hosts/hera.yaml

```diff
@@ -26,4 +26,6 @@ SUPPORTED_RESOLUTIONS: ['C1152', 'C768', 'C384', 'C192', 'C96', 'C48']
 COMINecmwf: /scratch1/NCEPDEV/global/glopara/data/external_gempak/ecmwf
 COMINnam: /scratch1/NCEPDEV/global/glopara/data/external_gempak/nam
 COMINukmet: /scratch1/NCEPDEV/global/glopara/data/external_gempak/ukmet
-AERO_INPUTS_DIR: /scratch1/NCEPDEV/global/glopara/data/gocart_emissions
+AERO_INPUTS_DIR: /scratch1/NCEPDEV/global/glopara/data/GEFS_ExtData/20250310
+DO_METP: 'NO'
```

Comment:
@DavidHuber-NOAA do we want this set to 'YES' on Hera?

Reply:
Oops. We can take that out.

Reply:
Thanks @bbakernoaa !